examples/tensorflow/nlp/bert_base_mrpc/quantization/ptq/README.md

```diff
@@ -62,129 +63,76 @@ bash prepare_model.sh --dataset_location=./data --output_dir=./model
-
-```yaml
-model:
-  name: bert
-  framework: tensorflow
-  inputs: input_file, batch_size
-  outputs: loss/Softmax:0, IteratorGetNext:3
-
-device: cpu                                          # optional. default value is cpu, other value is gpu.
-
-evaluation:
-  accuracy: {}
-  performance:
-    iteration: 20
-    warmup: 5
-    configs:
-      num_of_instance: 1
-      cores_per_instance: 28 
-      kmp_blocktime: 1
-
-quantization:            
-  calibration:
-    sampling_size: 500
-  model_wise:
-    weight:
-      granularity: per_channel
-  op_wise: {
-             'loss/MatMul': {
-               'activation':  {'dtype': ['fp32']},
-               'weight':  {'dtype': ['fp32']},
-             }
-           }
-tuning:
-  accuracy_criterion:
-    relative:  0.01   
-  exit_policy:
-    timeout: 0       
-    max_trials: 100 
-  random_seed: 9527
+For bert, we applied the first one as we already have write dataset and metric for bert mrpc task. 
+
+### Quantization Config
+The Quantization Config class has default parameters setting for running on Intel CPUs. If running this example on Intel GPUs, the 'backend' parameter should be set to 'itex' and the 'device' parameter should be set to 'gpu'.
 
 ```
-Here we set the input tensor and output tensors name into *inputs* and *outputs* field. In this case we calibrate and quantize the model, and use our calibration dataloader initialized from a 'Dataset' object.
+config = PostTrainingQuantConfig(
+    device="gpu",
+    backend="itex",
+    inputs=["input_file", "batch_size"],
+    outputs=["loss/Softmax:0", "IteratorGetNext:3"],
+    ...
+    )
+```
+Here we set the input tensor and output tensors name into *inputs* and *outputs* args. In this case we calibrate and quantize the model, and use our calibration dataloader initialized from a 'Dataset' object.
 
 ### Code update
 
 After prepare step is done, we add tune and benchmark code to generate quantized model and benchmark.
 
 #### Tune
 ```python
-      from neural_compressor.experimental import Quantization, common
-      quantizer = Quantization(FLAGS.config)
-      dataset = Dataset(eval_file, FLAGS.eval_batch_size)
-      quantizer.model = common.Model(estimator, input_fn=estimator_input_fn)
-      quantizer.calib_dataloader = common.DataLoader(dataset, collate_fn=collate_fn)
-      quantizer.eval_dataloader = common.DataLoader(dataset, collate_fn=collate_fn)
-      quantizer.metric = Accuracy()
-      q_model = quantizer.fit()
-      q_model.save(FLAGS.output_model)
+    from neural_compressor import quantization
+    from neural_compressor.config import PostTrainingQuantConfig
+    config = PostTrainingQuantConfig(
+        inputs=["input_file", "batch_size"],
+        outputs=["loss/Softmax:0", "IteratorGetNext:3"],
+        calibration_sampling_size=[500],)
```

Comment:
if sampling_size is single number, the underneath logic should be able to accept scaler but not only list. @PenghuiCheng